### ISSUE TITLE
Would be great to see this expanded to addon management..

### ISSUE THREAD

**Issue by XeonG** [unitycoder/UnityLauncherPro#55]
As we all know Unity built in asset package management system is a literal garbage tier effort.

Something where we can see what is already downloaded from the local folders provided, asset versions, and have the tool download and cache the images/description and more of the information that is found on the website that doesn't get cached in the built in package manager.. 

faster and better search, tagging, folders, publisher sorting etc.. would all make it so much better for looking over assets.

**Comment by unitycoder** [comment]
yup, several times a year it comes to mind that, what if could bypass package manager..
feel free to expand those ideas into quick sketches or lists (how it could/should work).

ideas:
at least my downloaded assets (locally) are easy to iterate from appdata folder, so that could another tab, "my assets"
and just shows list and info about those unity packages. (checking updates could be slow.. but i think no need to check for every package, just for the selected one, and could do it on demand only, if user clicks check updates for that)

**Comment by unitycoder** [comment]
adding link for reference, local package importer:
https://assetstore.unity.com/packages/tools/utilities/local-package-importer-105554

**Comment by unitycoder** [comment]
can open links into packagemanager:
asset store item, com.unity3d.kharma:content/1
UPM package links, com.unity3d.kharma:upmpackage/com.needle.engine-exporter
